src/input/Search.js

```diff
@@ -224,6 +243,17 @@ const styles = StyleSheet.create({
   },
   inputLight: {
     backgroundColor: colors.grey4
+  },
+  clearIcon: {
+    backgroundColor: 'transparent',
+    position: 'absolute',
+    right: 16,
+    top: 15.5,
```

Comment:
@Monte9 I copied it from the style of search icon. Do you still want me to change it?